### PR TITLE
Extra info about monitoring v0.1

### PR DESCRIPTION
**Pull request by chriscowley**

### CODE REVIEW

File: architecture/monitoring/README.md [hunk 1]
```diff
@@ -34,6 +39,31 @@ python ./prometheus-mvp.py
...
+## How exactly does Prometheus work?
+
+Prometheus itself does very little and is fairly UNIX-y in the way it works.
+Basically it supplies:
+
+- A Time-Series Database
+- A query language (PromQL) for that TSDB
+- Scheduling of "scrapes"
+- A rubbish web interface
+- An awesome API ta query he TSDB
```
Comment (bwitt) [inline architecture/monitoring/README.md:51]:
```suggestion
- An awesome API to query the TSDB
```

File: architecture/monitoring/README.md [hunk 2]
```diff
@@ -34,6 +39,31 @@ python ./prometheus-mvp.py
...
+
+Prometheus works on a "pull" model and expects the metrics to be available
+over HTTP in a specific format. Some applications talk this format natively
+such as Traefik, Etcd and RabbitMQ. Others (such as PostgreSQL, FreeIPA and
+**many** others) need what are called "Exporters" as a go-between.
+
+Alert rules are defined based on metrics queries and alerts are pushed to
+Alertmanager. ALertmanager takes care of routing the alert where neecessary.
+
+Promtheus does not worry about security itself. However, everythong is
```
Comment (bwitt) [inline architecture/monitoring/README.md:62]:
```suggestion
Promtheus does not worry about security itself. However, everything is
```

**Review by yezz123 — APPROVED** [review]

File: architecture/monitoring/README.md [hunk 3]
```diff
@@ -34,6 +39,31 @@ python ./prometheus-mvp.py
...
+- An awesome API ta query he TSDB
+- A separate module for routing alerts (Alertmanager)
+
+Prometheus works on a "pull" model and expects the metrics to be available
+over HTTP in a specific format. Some applications talk this format natively
+such as Traefik, Etcd and RabbitMQ. Others (such as PostgreSQL, FreeIPA and
+**many** others) need what are called "Exporters" as a go-between.
+
+Alert rules are defined based on metrics queries and alerts are pushed to
+Alertmanager. ALertmanager takes care of routing the alert where neecessary.
```
Comment (dpavlos) [inline architecture/monitoring/README.md:60]:
```suggestion
Alertmanager. Alertmanager takes care of routing the alert where necessary.
```

File: architecture/monitoring/README.md [hunk 4]
```diff
@@ -34,6 +39,31 @@ python ./prometheus-mvp.py
...
+
+Promtheus does not worry about security itself. However, everythong is
+simple HTTP, so liberal use of reverse-proxies is encouraged where TLS and/or
+authentication is necessary. In that case, Prometheus and Alertmanger
```
Comment (dpavlos) [inline architecture/monitoring/README.md:64]:
```suggestion
authentication is necessary. In that case, Prometheus and Alertmanager
```

File: architecture/monitoring/README.md [hunk 5]
```diff
@@ -46,9 +76,6 @@ use the same Grafana instance. ELK and Graylog are also worth considering.
 
 ## Responsiblities
```
Comment (dpavlos) [inline architecture/monitoring/README.md:77]:
```suggestion
## Responsibilities
```

File: architecture/monitoring/README.md [hunk 6]
```diff
@@ -46,9 +76,6 @@ use the same Grafana instance. ELK and Graylog are also worth considering.
...
 
-The monitoring team cannot realistically be responsible for how every single 
-is monitored. Prometheus has a huge library of exporters for almost everything.
-
 The monitoring team can be responsible for ensuring that the infrastructure is
 available to the application/infrastructure teams. Also that knowledge of how
 to be added to that infrastucture is suitably shared.
```
Comment (dpavlos) [inline architecture/monitoring/README.md:81]:
```suggestion
to be added to that infrastructure is suitably shared.
```

**Comment by hbjydev** [comment]
@NeilHanlon @nazunalika How accurate is this document still to how monitoring is done on our infra?

**Pull request closed without merging**